demo/index.html

```diff
@@ -159,6 +166,69 @@
           .catch(onError);
         });
 
+        startNotificationsButton.addEventListener('click', function() {
+          heartRateDevice.request().then(function(device) {
+            return heartRateMeasurement.startNotifications().then(function() {
+              template.text = device.name + ' Notifications session has started.';
+              progressBar.indeterminate = false;
+              heartRateMeasurement.addEventListener('value-changed', parseHeartRate);
+            })
+          })
+          .catch(onError);
+        });
+
+        stopNotificationsButton.addEventListener('click', function() {
+          heartRateDevice.request().then(function(device) {
+            return heartRateMeasurement.stopNotifications().then(function() {
+              template.text = device.name + ' Notifications session has stopped.';
+              progressBar.indeterminate = false;
+              heartRateMeasurement.removeEventListener('value-changed', parseHeartRate);
+            })
+          })
+          .catch(onError);
+        });
+
+        disconnectButton.addEventListener('click', function() {
+          heartRateDevice.disconnect();
+          batteryDevice.disconnect();
+          template.text = 'Bluetooth device(s) disconnected.';
+          progressBar.indeterminate = false;
+        });
+
+        function parseHeartRate(event) {
+          let data = event.target.value;
```

Comment:
The use of `let` without `'use strict';` makes Chrome prior to 49 unhappy. I get

`Uncaught SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode`

in the JS console when I click on the various buttons, which prevents the "Not Supported" message from showing up in the UI.